File: MonopolyDealGame/MonopolyDealGame/DrawPile.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace MonopolyDealGame
{
    class DrawPile:Deck
    {
        public DrawPile(Table gameTable, List<Card> cards)
        { 
        }

        public DrawPile(Table gameTable)
        {
        }

        public override string ToString()
        {
            return base.ToString();
        }
        public override String getDeckName() 
        {
            return "Draw Pile";
        }
    }
}

File: Monopoly Deal/src/cards/DrawPile.java
package cards;

import java.util.List;

import board.Table;

public class DrawPile extends Deck {

	@Override
	public String getDeckName() {
		return null;
	}

	public DrawPile(Table gameTable, List<Card> cards) {
	}

	public DrawPile(Table gameTable) {
	}

	public String toString() {
		return super.toString();
	}
}
